p2p/dht/bootstrap.go

```diff
@@ -64,6 +71,7 @@ BOOTLOOP:
 		case <-timeout.C:
 			return ErrFailedToBoot
 		case err := <-reschan:
+			i++
 			if err == nil {
 				return ErrFoundOurself
```

Comment:
as we talked offline - it is a valid case that a Lookup call on the local id will return the local node, it can happen if the local node is trying to bootstrap for the second time (in the case where it got killed after the first lookup). It is also possible to get itself in the first time the bootstrap is running. In such case, we should continue with the bootstrap process and not fail it.